### PR TITLE
improve keras/engine/saving.py

### PR DESCRIPTION
**Pull request by knightXun**
### Summary
1. rename argument: f as h5dict_instance
2. rename argument: f as groups
3. add some comments for save_weights_to_hdf5_group
### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [y] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

File: keras/engine/saving.py [hunk 1]
```diff
@@ -558,16 +558,25 @@ def load_attributes_from_hdf5_group(group, name):
...
 
-def save_weights_to_hdf5_group(f, layers):
+def save_weights_to_hdf5_group(group, layers):
+    """Saves weights into the HDF5 group.
+
+    # Arguments
+        group: A pointer to a HDF5 group.
+        layers: Layers to load.
+
+    # Returns
```
Comment (gabrieldemarmiesse) [inline keras/engine/saving.py]: Keras functions which don't return anything don't have a `# Returns` in their docstring.

File: keras/engine/saving.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
     h5py = None
 
 
-def _serialize_model(model, f, include_optimizer=True):
+def _serialize_model(model, h5dict_instance, include_optimizer=True):
```
Comment (fchollet) [inline keras/engine/saving.py]: An `_instance` suffix is not good style.

Comment (knightXun) [reply keras/engine/saving.py]: which one is better?

Comment (gabrieldemarmiesse) [reply keras/engine/saving.py]: I think `h5dict` is enough for a reader to understand what is going on.

Comment (fchollet) [reply keras/engine/saving.py]: Sounds good.

**Comment by gabrieldemarmiesse** [comment]
Any update on this?

**Comment by knightXun** [comment]
@gabrieldemarmiesse this test failed,but i dont know the reasons.

**Pull request merged**